instance/static/html/instance/appserver.html

```diff
@@ -47,11 +47,18 @@
     </table>
 
     <button ng-disabled="is_active || !appserver.is_healthy"
-            tooltip="This will update the DNS entries so that this app server is used for the instance. Only one app server can be active at any time."
-            tooltip-placement="right"
+            tooltip="This will add this app server to the load balancer pool for this instance."
+            tooltip-placement="below"
             ng-click="make_appserver_active()">
       Activate this app server
     </button>
+
+    <button ng-disabled="!is_active || instance.active_appservers.length < 2"
```

Comment:
We could allow disabling the last appserver as well to effectively take an instance offline, which is sometimes useful.  However, the potential damage from allowing this is probably bigger than what we gain, since we can always use the Django console to deactivate the last appserver.